src/runner/fixture-hook-controller.ts

```diff
@@ -99,12 +103,20 @@ export default class FixtureHookController {
 
         if (item) {
             const shouldRunBeforeHook = !item.started;
+            let success = false;
 
             item.started = true;
 
-            const success = shouldRunBeforeHook
-                            && await this._runFixtureBeforeHook(item, fixture.globalBeforeFn as Function, testRun)
-                            && await this._runFixtureBeforeHook(item, fixture.beforeFn as Function, testRun);
+            if (shouldRunBeforeHook) {
+                item.fixtureGlobalBeforeHookPromise = this._runFixtureBeforeHook(item, fixture.globalBeforeFn as Function, testRun);
+                success                             = await item.fixtureGlobalBeforeHookPromise;
+                item.fixtureBeforeHookPromise       = this._runFixtureBeforeHook(item, fixture.beforeFn as Function, testRun);
```

Comment:
You changed the behavior of hook calls here. The second hook should not be called if the first one failed